02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "derived_questions": [
+                    "Do clients have to list their adult children living with them if they have jobs and buy their own food?"
+                ],
+                "entire_card": "What if my client shares meals but wants to apply for food stamps on their own?\nIf a client expresses a desire to apply for SNAP separately from those they live and currently eat with, consider the following:\nIs the client sharing meals only temporarily? (due to immediate necessity or a temporary living situation)\nIs the client expressing a desire to use SNAP benefits only for themselves? (not other people they live with)\nIs the client/their household willing to provide a signed statement to the county office that their intention is to eat/prepare meals separately from others that they live with?\nIf any of the above are answered\nYES\nand the other household members are\nnot mandatory\nSNAP HH members, then the client can apply on their own. (Find the mandatory HH members here:\nWho are mandatory HH members for food stamps?\n)\nPLEASE NOTE\nWe should never lead or encourage a client to apply themselves disingenuously, nor should we suggest to a client that defining their SNAP household in one way versus another would be better for them. The purpose of this clarification is to acknowledge that SNAP applicants who actively express a desire to receive and use SNAP for themselves only can be applied in this manner. We should explain the rule and allow the client to apply how they desire.\nIf the processing office has concerns or uncertainties about the SNAP household, they may ask the applicant(s) to supply verification (like a signed statement).",
+                "summary": "Clients who express a desire to apply for SNAP separately from those they live and currently eat with may be able to do so if they meet certain criteria and provide verification if requested by the processing office."
+            },
+            "What if the only eligible household members are children under 18?": {
+                "score_sum": 0.3784036110580531,
+                "derived_questions": [
+                    "If they do have to list them, do we need to include the adult child's income?"
+                ],
+                "entire_card": "What if the only eligible household members are children under 18?\nIf all adults in the SNAP household are ineligible non-citizens, but have children who are eligible, the adult will still need to be listed as the head of household. Minors can not be listed as head of household.\nThe income/resources of the adult is still counted in these situations. The system will know how to count it to accurately determine eligibility.",
+                "summary": "In cases where all adults in the SNAP household are ineligible non-citizens but have eligible children, the adult must still be listed as the head of household and their income/resources will be counted for determining eligibility."
```

Comment:
**Relevance to the question**: 🔴 Irrelevant

**Correctness as a summary of the Guru card**: 🟢 100% correct

**Helpfulness**: 🟠 Unhelpful